chrooted.sh

```diff
@@ -0,0 +1,25 @@
+set -ex
+
+# Install the packages we need in the ipa-ramdisk
+dnf install -y \
+openstack-ironic-python-agent \
+python3-bcrypt \
```

Comment:
Is it a temporary workaround? Let's mark with FIXME until python3-ironic-lib is updated?

Reply:
yes, I'll add a note